app/src/main/java/com/babylon/wallet/android/presentation/common/UiMessage.kt

```diff
@@ -85,4 +91,22 @@ sealed class UiMessage(val id: String = UUIDGenerator.uuid().toString()) {
             }
         }
     }
+
+    data class GoogleAuthErrorMessage(
+        private val error: Throwable
+    ) : UiMessage() {
+
+        @Composable
+        override fun getMessage(): String { // TODO update copies accordingly
```

Comment:
do you think we will go as granular as explaining each of those errors? Maybe just one error message specific to google auth will be enough?